docker-compose.yml

```
@@ -83,3 +83,5 @@ services:
       /usr/bin/mc mb --quiet s3local/mint-app-file-uploads/;
       /usr/bin/mc policy set public s3local/mint-app-file-uploads;
       "
+  email:
+    image: dockage/mailcatcher:0.7.1
```

```suggestion
    image: dockage/mailcatcher:0.7.1
        ports:
      - 1080:1080
```

When I tested, I couldn't access mail catcher without adding this port mapping. With it, I can access the web UI. Mind adding that line in?

---

The [easi-app definition for MailCatcher](https://github.com/CMSgov/easi-app/blob/main/docker-compose.override.yml#L68-L71) also forwards port `1025`, which I think is the default SMTP port for mailcatcher. Might not be necessary, but is nice to have if you want to send an email to mailcatcher from oustide the docker-compose network!

---

that's a good point @ClayBenson94! It's a good thing to keep in mind for the future